src/rprblender/properties/render.py

```diff
@@ -429,7 +429,7 @@ def is_tile_render_available(self):
         ]
     if pyrpr2.enabled:
```

Comment:
With config variable removal this check produces error since pyrpr2.enabled is False.
It seems this check should be removed.